frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../variables.scss";
+ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  width: 100vh;
+}
+li {
+  color: $primaryColor;
+  list-style: none;
+  border: 2px solid $secondaryColor;
+  text-align: center;
+  position: relative;
+}
+.card {
+  height: 100vh;
+  background-size: cover;
+  width: 20vw;
+}
+button {
+  background-color: $primaryColor;
+  font-size: 1.3rem;
+  font-weight: bold;
+  width: 10vw;
+  height: 3vw;
+  border-radius: 10% 30% 50% 70%;
+  a {
+    text-decoration: none;
+    color: $secondaryColor;
+  }
+}
+h2 {
+  margin-bottom: 35rem;
```

Comment:
Why such a low margin? Is it responsive?

Reply:
done